### DISCUSSION TITLE
Add more control for new welcome/home screen in v1.0.x

### DISCUSSION THREAD

**Discussion by kamalkeshavani-aiinside** [apache/superset#18453] [Ideas]
**Is your feature request related to a problem? Please describe.**
The new Welcome/Home screen of Superset v1.0.x includes Recents, Dashboards, Saved Queries & Charts sections.
![image](https://user-images.githubusercontent.com/74634977/109264692-b8e62100-7848-11eb-9e7e-c85bbfbcd54b.png)

For many users in our case, they only use Superset to look at dashboards. So these users don't know about Charts/Saved Queries, as we have even removed those menu items from UI.

**Describe the solution you'd like**
We would like to control the sections visible to users on Welcome/Home screen based on role permissions, like we can control the Menu items in UI for specific roles using `menu access on ....` permission.

**Describe alternatives you've considered**
If this feature can take time, then I suggest to atleast make the minimize action of any Section on Home screen by user as sticky along with [#13303](https://github.com/apache/superset/issues/13303) 
Or maybe both can be implemented.

**Additional context**

**Comment by kanchansapkota27** [comment]
I also have similar requirements.
